sdk/servicebus/service-bus/src/core/receiverHelper.ts

```diff
@@ -37,9 +37,10 @@ export class ReceiverHelper {
   }
 
   /**
-   * Prevents us from receiving any further messages.
+   * Drains the receiver, preventing any new activity by
+   * causing addCredits to no longer add any credits.
```

Comment:
This was slightly confusing for me on the first read, can we reword as below?

```suggestion
   * Drains the receiver and prevents the `addCredit()` method from adding credits.
   * Call `unsuspend()` to enable the `addCredit()` method.
```

Reply:
Updated!